docs/self-managed/identity/what-is-identity.md

```diff
@@ -5,7 +5,17 @@ sidebar_label: "What is Identity?"
 description: "Identity is the component within the Camunda 8 stack responsible for authentication and authorization."
 ---
 
-Identity is the component within the Camunda 8 stack responsible for authentication and authorization. It allows you to manage:
+Identity is responsible for authentication and authorization within the Camunda 8 stack. Identity functions differently in Orchestration clusters and in Console and Web Modeler. For more information on these differences, see the Self-Managed [reference architecture](/self-managed/reference-architecture/reference-architecture.md#orchestration-cluster-vs-web-modeler-and-console).
```

Comment:
🔧  we should indicate these are different identity components indeed
```suggestion
Identity components are responsible for authentication and authorization within the Camunda 8 stack. There are two different identity components in the stack, one as an integral part of the Orchestration cluster and spanning across orchestration clusters used for Console and Web Modeler access management. For more information on these differences, see the Self-Managed [reference architecture](/self-managed/reference-architecture/reference-architecture.md#orchestration-cluster-vs-web-modeler-and-console).
```

Reply:
I've tried to clarify this further in the latest update! Once there is a place in the docs for Orchestration cluster docs vs Everything else docs, this page will mostly be able to go away, and I hope this distinction will feel much more separate.